src/addons/TransitionablePortal/TransitionablePortal.js

```diff
@@ -99,13 +97,8 @@ export default class TransitionablePortal extends Component {
 
   handlePortalClose = () => {
     debug('handlePortalClose()')
-    const { open } = this.props
-    const { portalOpen } = this.state
 
-    // Heads up! We simply call `onClose` when component is controlled with `open` prop.
```

Comment:
Cruft, we don't call `onClose` and it seems that condition below doesn't make sence.